osu.Game/Users/Drawables/DrawableFlag.cs

```diff
@@ -29,7 +30,8 @@ private void load(TextureStore ts)
             if (ts == null)
                 throw new ArgumentNullException(nameof(ts));
 
-            Texture = ts.Get($@"Flags/{country?.FlagName ?? @"__"}") ?? ts.Get(@"Flags/__");
+            string textureName = country == default ? "__" : country.ToString();
+            Texture = ts.Get($@"Flags/{textureName}") ?? ts.Get(@"Flags/__");
```

Comment:
Could alternatively rename `__.png` in resources project to `XX.png` rather than doing this, but didn't want to initiate another game resources release just for this.